src/Pages/Dashboard/ShowHistoryGraph.jsx

```diff
@@ -0,0 +1,123 @@
+import React, { PureComponent } from "react";
+import {
+  Paper,
+  Typography,
+  FormControl,
+  InputLabel,
+  Select,
+  MenuItem,
+  Grid,
+  withStyles,
+} from "@material-ui/core"
+import {
+  DateTimePicker,
+  MuiPickersUtilsProvider,
+} from '@material-ui/pickers';
+import DateFnsUtils from '@date-io/date-fns'; // choose your lib
+import { historyStyle, materialTheme } from "./dashboardStyle";
+import ChartContainer from "./ChartContainer";
+import ThemeProvider from "@material-ui/styles/ThemeProvider";
+
+class ShowHistoryGraph extends PureComponent {
+  constructor(props) {
+    super(props);
+    this.state = {
+      data: null,
+    };
+  }
+
+  render() {
+    const { classes } = this.props;
+    const now = Date.now();
+
+    this.selected = this.props.selected;
+    this.start = this.props.start;
+    this.end = this.props.end;
+    this.changeRange = this.props.changeRange;
+    this.historyData = this.props.historyData;
+    this.changeCalStart = this.props.changeCalStart;
+    this.changeCalEnd = this.props.changeCalEnd;
+
+    return (
+      <Grid container spacing={3}>
+        <Grid item xs={6}>
+          <Grid>
+            <Paper className={classes.paper}>
+              <Typography variant="h6" color="primary" gutterBottom>
+                Date Range
+              </Typography>
+              <FormControl style={{ minWidth: 120 }} className={classes.formControl}>
+                <InputLabel shrink htmlFor="range" className={classes.inputLabel}>Range</InputLabel>
+                <Select
+                  name="range-select"
+                  className={classes.select}
+                  inputProps={{
+                    classes: {
+                      icon: classes.icon,
+                    },
+                  }}
+                  autoWidth={[ true ]}
```

Comment:
I think `autoWidth` takes a boolean rather than array.

Reply:
Do I just move the array then?